stl/inc/xmemory

```diff
@@ -2003,6 +2007,99 @@ typename _Container::size_type _Erase_nodes_if(_Container& _Cont, _Pr _Pred) {
     }
     return _Old_size - _Cont.size();
 }
+
+#if _HAS_CXX20
+template <class _Ty, class _Alloc, class... _Types, enable_if_t<!_Is_specialization_v<_Ty, pair>, int> = 0>
+constexpr auto uses_allocator_construction_args(const _Alloc& _Al, _Types&&... _Args) noexcept {
+    if constexpr (!uses_allocator_v<_Ty, _Alloc>) {
+        static_assert(is_constructible_v<_Ty, _Types...>,
+            "T must be constructible from Types... if uses_allocator_v<T, Alloc> is false");

```

Comment:
I would flip the wording, the most relevant information here is `uses_allocator_v` is false
```suggestion
            "If uses_allocator_v<T, Alloc> does not hold, T must be constructible from Types... ");
```